enterprise/views.py

```diff
@@ -555,10 +555,13 @@ def create_enterprise_course_enrollment(request, enterprise_customer, course_id,
             enterprise_customer=enterprise_customer,
             user_id=request.user.id
         )
+        enterprise_enrollment_source = EnterpriseEnrollmentSource.get_source(
+            EnterpriseEnrollmentSource.ENROLLMENT_URL)
```

Comment:
I think we usually put closing parentheses on new lines... unless something's changed